functional/vfio/run.sh

```diff
@@ -20,7 +20,16 @@ tmp_data_dir="$(mktemp -d)"
 trap cleanup EXIT
 
 CONFIG_FILE="${tmp_data_dir}/configuration.toml"
+
+# kata-runtime options
 SANDBOX_CGROUP_ONLY=""
+hypervisor=
+machine_type=
+
+# config option paths and strings
+config_options_path="/usr/share/defaults/kata-containers"
```

Comment:
this variable is not needed, you can get this from `kata-runtime --kata-show-default-config-paths`